lib/material/material_gui.flow

```diff
@@ -372,6 +372,10 @@ export {
 	// Identical to MFrame, only the frame is placed on top of the "content"
 	MOverlay(margin : double, radius : double, style : [MGraphicsStyle], content : Material) -> Material;
 	MOverlayT(margin : Transform<double>, radius : Transform<double>, style : Transform<[MGraphicsStyle]>, content : Material) -> Material;
+
+	//Helper for Segmented buttons
```

Comment:
add the link here pls https://m3.material.io/components/segmented-buttons/overview